CMakeLists.txt

```diff
@@ -14,53 +14,30 @@ SET(CMAKE_CXX_STANDARD_REQUIRED ON)
 SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}  -w")
 SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -w")
 
-SET(BUILD_TESTS TRUE CACHE BOOL "Build engine tests")
-
+SET(BUILD_TESTS TRUE CACHE BOOL "Build OpenMapper tests")
+SET(BUILD_APP FALSE CACHE BOOL "Build OpenMapper App")
+SET(BUILD_IOS_FW FALSE CACHE BOOL "Build OpenMapper iOS Framework")
 
 ################################
-# File Globbing
+# Define Sources and Headers
 ################################
 
 FILE(GLOB_RECURSE OM_LIB_HEADERS "${CMAKE_CURRENT_LIST_DIR}/include/wrapper.h")
 FILE(GLOB_RECURSE OM_LIB_SOURCES "${CMAKE_CURRENT_LIST_DIR}/src/wrapper.cc")
 
-SET(OM_LIB_RESOURCES ${OM_LIB_SOURCES} ${OM_LIB_HEADERS})
-
+IF(BUILD_IOS_FW)
+  SET(OM_LIB_RESOURCES ${OM_LIB_SOURCES} "${CMAKE_CURRENT_LIST_DIR}/wrapper/ios/open_mapper.mm"
+                                         "${CMAKE_CURRENT_LIST_DIR}/wrapper/ios/open_mapper.h")
+ENDIF(BUILD_IOS_FW)
 SET(OM_APP_RESOURCES "${CMAKE_CURRENT_LIST_DIR}/src/app/main.cc")
 
-LIST(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake_modules)
-
-################################
-# Target Definitions
-################################
+# LIST(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake_modules)
```

Comment:
This seems to be important for the moment for travis. This will eventually not be needed anymore if we change to the eigen3 binaries as ros package. (As we did with openCV3)